oz/core/actions.py

```diff
@@ -107,7 +119,15 @@ def server():
             signal.signal(signal.SIGTERM, functools.partial(_shutdown_handler, http_srv))
             signal.signal(signal.SIGINT, functools.partial(_shutdown_handler, http_srv))
 
-        tornado.ioloop.IOLoop.instance().start()
+        if server_type == "asyncio":
```

Comment:
All non-tornado server types will not work with the shutdown handler because it explicitly calls `tornado.ioloop.IOLoop.instance()`. This should be noted, or we should update the shutdown handler to handle the other event loop types.